cmd/statusd/main.go

```diff
@@ -33,6 +33,21 @@ var (
 	logLevel       = flag.String("log", "INFO", `Log level, one of: "ERROR", "WARN", "INFO", "DEBUG", and "TRACE"`)
 	logFile        = flag.String("logfile", "", "Path to the log file")
 	version        = flag.Bool("version", false, "Print version")
+	les            = flag.Bool("les", false, "LES protocol")
+
+	listenAddr   = flag.String("listenaddr", ":30303", "IP address and port of this node (e.g. 127.0.0.1:30303)")
+	identity     = flag.String("shh.identity", "", "Protocol identity file (private key used for asymmetric encryption)")
```

Comment:
To align with `passwordFile` maybe we could call it `identityFile`?

Reply:
seems with that change only ... may result in some confusion, since flag identity is set to a similarly named field in cmd/statusd/whispercfg.go:
      whisperConfig.IdentityFile = *identity

Reply:
I am not sure if I understand. So, if we change `identity` flag to `identityFile` and then it will be assigned as `whisperConfig.IdentityFile = *identityFile` it should not introduce any confusion.

Reply:
okay I was thinking you meant changing passwordfile to indentityfile ... 
I'll change identity to indentityfile in cli and identifyFile as go var:
 identityFile = flag.String("shh.identityfile", ...